src/main/java/com/spotify/reaper/resources/RepairRunResource.java

```diff
@@ -161,12 +161,10 @@ public Response addRepairRun(
         parallelism = RepairParallelism.valueOf(repairParallelism.get().toUpperCase());
       }
       
-      if(!parallelism.equals(RepairParallelism.PARALLEL) && incrementalRepair) {
-          return Response.status(Response.Status.BAD_REQUEST).entity(
-                  "It is not possible to mix sequential repair and incremental repairs. parallelism " 
-                  + parallelism + " : incrementalRepair " + incrementalRepair).build();
+      if (incrementalRepair) {
+        parallelism = RepairParallelism.PARALLEL;
```

Comment:
Should we just step on the parallelism option or return with a validation error when incremental and parallel are true? Either way, we should add in to javadoc saying we change it.